### DISCUSSION TITLE
为什么总是会自动更新

### DISCUSSION THREAD

**Discussion by MwisQing** [the1812/Bilibili-Evolved#1305] [提问 / Q&A]
用的qq浏览器，知道1.15有不适配，所以用的1.11，把@updateURL  、@downloadURL 替换为空了之后还是会自动更新

**Comment by the1812** [comment]
是离线版吗

**Comment by MwisQing** [comment]
是离线版，现在好像又没有自动更新了



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Grant Howard"<notifications@github.com&gt;; 
发送时间: 2020年12月12日(星期六) 上午10:43
收件人: "the1812/Bilibili-Evolved"<Bilibili-Evolved@noreply.github.com&gt;; 
抄送: "丿小少爷彡"<562245610@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [the1812/Bilibili-Evolved] 为什么总是会自动更新 (#1305)





 
是离线版吗
